src/Action/NotifyAction.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusQuickpayPlugin\Action;
+
+use Payum\Core\Action\ActionInterface;
+use Payum\Core\ApiAwareInterface;
+use Payum\Core\Exception\RequestNotSupportedException;
+use Payum\Core\GatewayAwareInterface;
+use Payum\Core\GatewayAwareTrait;
+use Payum\Core\Reply\HttpResponse;
+use Payum\Core\Request\GetHttpRequest;
+use Payum\Core\Request\Notify;
+use Safe\Exceptions\JsonException;
+use Setono\Payum\QuickPay\Action\Api\ApiAwareTrait;
+use Setono\Payum\QuickPay\Model\QuickPayPayment;
+use Sylius\Component\Core\Model\PaymentInterface;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+
+class NotifyAction implements ActionInterface, ApiAwareInterface, GatewayAwareInterface
```

Comment:
How does Payum know to use this instead of the one in the `payum-quickpay` lib?

Reply:
Different models
https://github.com/vvasiloi/SyliusQuickpayPlugin/blob/aa894ab7ee48454e4597861e01096d2d2e379782/src/Controller/NotifyAction.php#L82-L83
https://github.com/vvasiloi/SyliusQuickpayPlugin/blob/aa894ab7ee48454e4597861e01096d2d2e379782/src/Action/NotifyAction.php#L61

Reply:
It's similar to how the message buses work, I know you use these a lot.

Reply:
I understand. Great